### PR TITLE
Change "le met" to "let me"

### PR DESCRIPTION
**Pull request by MysteriousLog6**
Fixed spelling mistake

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -124,7 +124,9 @@ Or::
 
     $ man budget
 
-if you have installed the man pages.
```
Comment (wichtounet) [inline README.rst:127]: This change is not necessary.

**Review by wichtounet — CHANGES_REQUESTED** [review]
Good catch for the let me, but there is an extra change in your PR that is not necessary.

File: README.rst (hunk 1 above)
Comment (MysteriousLog6) [reply README.rst:127]: Sorry! I didn't notice it 😂

**Review by wichtounet — APPROVED** [review]

**Comment by wichtounet** [comment]
Thanks for the change 👍

**Pull request merged**